Updated css and content

rajbirjohar opened the pull request:
**Changed**

- converted hero background image into just a background image for h1 (this is just an experiment. we can always change it back if there is creative conflict) and background image for viruses. Created parallax effect (I just thought it looked cool and might add some flair to the pages)
- removed helix in the navbar
- Changed navbar to white but its blended into navbar so I'm trying to figure out how to style it beyond just background color
- Removed slack link
- Added updated announcements 
- Converted p font to raleway but kept open sans for h1
- Added navbar shadow
- Added active link animation to background of navbar links so users know what page they are on. Does not currently work for home page.

**Suggestions**

- Try to find a way to change the color of navbar based on page
- Drop shadow navbar after scrolling

rajbirjohar committed:
Merge pull request #24 from jaeeee/master

Update branch-RJ

rajbirjohar committed:
Merge pull request #25 from jaeeee/master

Update branch-RJ

r-jo1 committed:
Updated css

- Turned full background into card background for hero and virus and added parallax effect
- Converted p font into raleway but kept open sans for h1
- removed slack link
- added updated announcment

r-jo1 committed:
colored the navbar button grey just for place holder

johan1505 commented:
Feedback:

- Make announcements a list of bullet points
- The dropdown button blends too much with the white navigation bar. I think is better to keep it as orange
-  Shadow on the navigation bar and active link animation look good
- Background image for viruses looks good on mobile, but on desktop, the page feels unfilled. Same for the main title on the home page

r-jo1 committed:
Updated css

- Increased size of hero background
- increased font size from 14pt to 16pt
- Added list in announcement

r-jo1 committed:
Updated button color

- Converted from navbar-dark to navbar-light
- removed border

rajbirjohar commented:
> * Make announcements a list of bullet points

I converted it to open circle list

> * The dropdown button blends too much with the white navigation bar. I think is better to keep it as orange

I removed the border and converted it to a dark color to see better on the white

> * Background image for viruses looks good on mobile, but on desktop, the page feels unfilled. Same for the main title on the home page

I increased the size of the background on the hero page but I'm not sure how to fill up the other pages asides from increasing font size.
